test/TensorFlowRuntime/top_level_2.swift

```diff
@@ -1,5 +1,5 @@
 // RUN: %target-run-simple-swift %swift-tensorflow-test-run-extra-options
```

Comment:
please document that we are changing the semantics of `%target-run-simple-swift` from running GPE to eager in this PR.

It might be slightly better to instead call sth like `run-eager-swift` or `run-dynamic-compilation-swift`, so that if we ever flip graph mode back to true again, we need not do another round of test header updates. You can decide.

Reply:
Actually, run-eager-swift seems like a good idea. Done. Changed the tests to use "run-eager-swift" instead of "run-simple-swift"